zk-token-sdk/src/instruction/mod.rs

```diff
@@ -19,6 +20,9 @@ use {
 };
 pub use {
     bytemuck::Pod,
+    ctxt_comm_equality::{
```

Comment:
Same comment as on the other PR about spelling things out. Since this is a new module, can we go ahead and spell out ciphertext_commitment_, please?
(My brain wants to read it as context_communication as it is...)

Reply:
Okay, will do!